datalake/backend/endpoints/dataset_versions/create.py

```diff
@@ -0,0 +1,106 @@
+"""Import dataset-version function."""
+import json
+import logging
+import uuid
+from os import environ
+
+import boto3
+from jsonschema import ValidationError, validate  # type: ignore[import]
+from pynamodb.exceptions import DoesNotExist
+
+from ..datasets.model import DatasetModel
+from ..utils import DATASET_TYPES, ENV, JSON_OBJECT, error_response, success_response
+
+stepfunc_client = boto3.client("stepfunctions")
+ssm_client = boto3.client("ssm")
+
+STEP_FUNCTION_ARN_PARAMETER_NAME = f"/{ENV}/StepFuncStateMachineARN"
+
+
+def create_dataset_version(payload: JSON_OBJECT) -> JSON_OBJECT:
+    """POST: Create Dataset."""
```

Comment:
This docstring should be changed to "Create Dataset version"

Reply:
Why not remove this docstring? The only thing it tells us in addition to the function name is just that it's responding to a POST request (which the contents of the function doesn't actually care about).